### ISSUE TITLE
Otel question

### ISSUE THREAD

**Issue by pdgonzalez872** [mtrudel/bandit#358]
Hi @mtrudel ! Thanks for Bandit!

I've been using it in production for a couple of months and things have worked well!

Quick question about opentelemetry: I see this lib on hex: https://github.com/samokat-oss/opentelemetry-bandit and was wondering if this is the way we should use otel with bandit?

Here is a commit that adds the above to the opentelemetry getting started guide: https://github.com/open-telemetry/opentelemetry.io/commit/d02ad9dbd66336302cb1e7831741e971035cf5cf. I wanted to check with you first before opening a PR to add it.

If this is **not** the way, what should we use?

Thanks again!

**Comment by pdgonzalez872** [comment]
I just saw this: https://github.com/open-telemetry/opentelemetry-erlang-contrib/tree/main/instrumentation/opentelemetry_bandit.

Interesting! Maybe it's just a typo in the Github link on hex? That's how I got to that personal repo.

https://hex.pm/packages/opentelemetry_bandit _should_ resolve to the `opentelemetry-erlang-contrib` one I think, so the instructions in the commit are valid, but there could be a typo somewhere :)
